src/helpers/pythonToFrames.ts

```diff
@@ -9,6 +9,8 @@ import FrameContainer from "@/components/FrameContainer.vue";
 
 const TOP_LEVEL_TEMP_ID = -999;
 
+const parsedTripleQuotesStrRegex = /^''.*''$/s;
```

Comment:
Maybe make a comment here about why there are only two quotes: I think it's because this is the string content once the outer quotes are processed by Skulpt?  Also, what about triple double-quote strings, i.e. """ blah """ ?